cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -93,11 +96,17 @@ func TestEvaluateExecute(t *testing.T) {
 				tc.ExpectedOutputValidate(t, logOutput.String(), temporaryPath)
 			}
 
-			actualResultFiles, err := osutil.FilesRecursive(temporaryPath)
+			actualResultFilesRaw, err := osutil.FilesRecursive(temporaryPath)
 			require.NoError(t, err)
-			for i, p := range actualResultFiles {
-				actualResultFiles[i], err = filepath.Rel(temporaryPath, p)
+			actualResultFiles := make([]string, 0, len(actualResultFilesRaw))
+			for _, p := range actualResultFilesRaw {
+				if strings.Contains(p, "TESTIGNORE") {
```

Comment:
What is that?

Reply:
Ah, this is to ignore the ollama model directory. I think it is better to just add subdirectories to the temporary directory, because then you always have a subdirectory that is clean of anything else.